src/diffusers/pipelines/wan/pipeline_wan_i2v.py

```diff
@@ -49,11 +49,11 @@
         >>> import numpy as np
         >>> from diffusers import AutoencoderKLWan, WanImageToVideoPipeline
         >>> from diffusers.utils import export_to_video, load_image
-        >>> from transformers import CLIPVisionModel
+        >>> from transformers import CLIPVisionModelWithProjection
 
         >>> # Available models: Wan-AI/Wan2.1-I2V-14B-480P-Diffusers, Wan-AI/Wan2.1-I2V-14B-720P-Diffusers
         >>> model_id = "Wan-AI/Wan2.1-I2V-14B-480P-Diffusers"
-        >>> image_encoder = CLIPVisionModel.from_pretrained(
+        >>> image_encoder = CLIPVisionModelWithProjection.from_pretrained(
```

Comment:
why this change? the authors just changed to CLIPVisionModel https://github.com/huggingface/diffusers/pull/10975/files